src/main/java/edu/harvard/iq/dataverse/export/ddi/DdiExportUtil.java

```diff
@@ -946,9 +1064,10 @@ private static void writeDistributorsElement(XMLStreamWriter xmlw, DatasetVersio
                                 if (!distributorURL.isEmpty()) {
                                     writeAttribute(xmlw, "URI", distributorURL);
                                 }
-                                if (!distributorLogoURL.isEmpty()) {
+                                /* NOT IN THE SCHEMA! -L.A.if (!distributorLogoURL.isEmpty()) {
+                                   (and why were we putting this logo into the "role" field anyway?? - same with producerLogo above!)
```

Comment:
Does it make sense to remove the producer logo URL now with these other breaking changes? No strong opinion, but if it really isn't useful, it might be easier to change now than in a later PR.

Reply:
Just in case it's helpful: I [wrote about this a bit](https://docs.google.com/document/d/1ICXRL8DP5fCGYiRyRphh_3OotNaWOOak1VmnyufBNsM/edit#bookmark=id.lhydbw2njf43) when I tried to help in 2017. Back then I guessed that the distributor logo url was put into this "role" field "to preserve that metadata during migrations from Dataverse 3.x to 4.x".

Reply:
@qqmyers @jggautier Thank you for the comments. I've been meaning to finalize this - after everything else is fixed, that is. But hoping to wrap this up first thing next week. 
I'm leaning towards the simplest/brutest force solution that Jim mentioned - just dropping this "role" attribute, since the logo url makes zero sense in it. 
Julian, thank you for the writeup. I dug a bit further, by reading the old DVN code. And it doesn't look like it was done for the purposes of the migration either. It seems like this is even more ancient history, that it may be tracing its history to a hack that DVNs used to rely on to pass the logos when harvesting from each other. However, unless I'm really not reading it right, it just doesn't seem like it was doing the same thing as the current Dataverse exporter - it was never putting that logo url in the "role" attribute; it was encoding it as an HTML link inside the text of the producer field, with a "role=..." attribute of its own... But then, when the export and import code were being ported to Dataverse, this may have been changed simply by mistake (?). Does this make any sense? - it doesn't make complete sense to me, I feel like I may be missing something - but I am fairly confident by now that nobody could possibly be relying on that attribute in its current form now. So I am indeed feeling like killing it.

Reply:
@kcondon I was able to confirm that both of the "all fields" json examples that we provide in the source tree: `./src/test/java/edu/harvard/iq/dataverse/export/ddi/dataset-create-new-all-ddi-fields.json` and `scripts/api/data/dataset-create-new-all-default-fields.json` are now importable (I applied your fixes to both of the above in my branch). And, both of the resulting datasets are exportable as ddi, and the produced ddi is valid. Tested both in my build env. and on dataverse-internal (deployed my build there). 
The dataset you showed me earlier there, `FK2/JA1SDI` is still failing to export, with the same misleading error message. You mentioned that it was produced by exporting an existing dataset produced from one of the sources above in json and re-importing... so I'm guessing something funny gets added or changed during that roundtrip. I'm going to investigate that next.

Reply:
Just confirming that I have dropped this dubious "role=logo" attribute from the export.